numpy/core/src/umath/ufunc_object.c

```diff
@@ -4671,6 +4853,7 @@ PyUFunc_FromFuncAndDataAndSignature(PyUFuncGenericFunction *func, void **data,
     ufunc->core_dim_ixs = NULL;
     ufunc->core_offsets = NULL;
     ufunc->core_signature = NULL;
+    ufunc->core_dim_szs = NULL;
```

Comment:
Should also initialize `core_dim_flexible`

Reply:
fixed